ResearchKit/Common/ORKResult.m

```diff
@@ -1226,6 +1260,21 @@ - (id)answer {
     return nil;
 }
 
+- (NSString *)descriptionWithNumberOfPaddingSpaces:(NSUInteger)numberOfPaddingSpaces {
+    NSMutableString *description = [NSMutableString stringWithFormat:@"%@; answer:", [self descriptionPrefixWithNumberOfPaddingSpaces:numberOfPaddingSpaces]];
+    id answer = self.answer;
+    if ([answer isKindOfClass:[NSArray class]]
+        || [answer isKindOfClass:[NSDictionary class]]
+        || [answer isKindOfClass:[NSSet class]]
+        || [answer isKindOfClass:[NSOrderedSet class]]) {
+        [description appendFormat:@"\n%@\n%@>", answer, ORKPaddingWithNumberOfSpaces(numberOfPaddingSpaces)];
```

Comment:
Agreed, but I'm not sure how to do that effectively.

From outside, you can only control indentation of the first line when calling `description` on collection instances (i.e., `NSArray`, `NSDictionary`, `NSSet`, and `NSOrderedSet`).

A way to do this that I can think of is to implement `ork_descriptionWithNumberOfPaddingSpaces:` in extensions for the collection types, which would manually iterate the contents and avoid using `description`.

What do you think? Should I try this approach or do you have any more elegant solution in mind?